python/nnvm/top/tensor.py

```diff
@@ -37,11 +36,42 @@ def _compute(attrs, x, _):
         return f(x[0], x[1])
     return _compute
 
-
 _fschedule_injective = tvm.convert(_schedule_injective)
 _fschedule_broadcast = _fschedule_injective
 _fschedule_elemwise = _fschedule_injective
 
+# lshift
+@reg.register_compute("left_shift")
+def compute_lshift(attrs, inputs, _):
+    data = inputs[0]
+    bit = attrs.get_int("bit")
+    return topi.left_shift(data, bit)
+reg.register_pattern("left_shift", OpPattern.ELEMWISE)
+reg.register_schedule("left_shift", _fschedule_broadcast)
+
+# rshift
+@reg.register_compute("right_shift")
```

Comment:
this is not consistent with numpy left_shift. To make it consistent with current add, sub, we should name it ```__rshift_scalar__```